nncf/compression_method_api.py

```diff
@@ -51,14 +51,20 @@ class PTCompressionLoss(nn.Module, CompressionLoss):
     sparsity algorithm calculates the number of non-zero weights in convolutional
     and fully-connected layers to construct the loss function.
     """
+    def __init__(self, default_device=None):
+        """
+        :param default_device: torch device being used for computing internal tensors
```

Comment:
it's not actually for computing, it's just for creating a default value of loss on the target device